drivers/display/Kconfig.ssd1322

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Lukasz Hawrylko
+# SPDX-License-Identifier: Apache-2.0
+
+config SSD1322
+	bool "SSD1322 display driver"
+	default y
+	depends on DT_HAS_SOLOMON_SSD1322_ENABLED
+	select MIPI_DBI
+	help
+	  Enable driver for SSD1322 display driver.
+
+if SSD1322
+
+config SSD1322_BUFFER_SIZE
+	int "SSD1322 internal buffer size"
+	default 8
+	range 8 61440
+	help
+	  Size of the SSD1322 internal pixel format conversion buffer, in bytes.
+	  Maximum value is the size of the display ram, 480*128=61440.
```

Comment:
There is already information in devicetree what buffer size is needed. So you could add
```c
struct ssd1322_data {
    uint8_t *const buf;
}
```
and define a buffer based on `DT_PROP(node_id, height)....` and assign it to the pointer. This will also give you better multi-instance support. Also maximum size is to much, good compromise should be buffer large enough for line.